### PR TITLE
Sync time of RF Daemon every 30min

### PR DESCRIPTION
**Pull request by IceOnly**
Feature: #421

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @IceOnly, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;hassio-addons#425).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: homematic/run.sh [hunk 1]
```diff
@@ -75,5 +75,15 @@ if [ "$WIRED_ENABLE" == "true" ]; then
...
 
+# sync time periodically
+if [ "$RF_ENABLE" == "true" ]; then
+    while true
+    do
+        sleep 30m
+        echo "Run SetInterfaceClock"
+        "$HM_HOME/bin/SetInterfaceClock" 127.0.0.1:2001
+    done
+fi
```
Comment (pvizeli) [inline homematic/run.sh]: That run now in a endless loop and don't handle the docker events. Can you add a signal handler function they for SIGTERM, SIGHUB?

https://github.com/home-assistant/hassio-addons/blob/master/samba/run.sh

Comment (IceOnly) [reply homematic/run.sh]: of course, I will add them at the weekend.
I was thinking sleep will be interrupted from SIGTERM and SIGKILL like the wait operation at the end of the script.

Comment (IceOnly) [reply homematic/run.sh]: @pvizeli can you tell me what you mean with " Don't handle the docker events.".
I can stop and start the container without problems.

**Comment by pvizeli** [comment]
Looks good

**Pull request merged**